WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -254,7 +257,7 @@ public void onClick(View v) {
             }
         });
 
-        rootView.findViewById(R.id.row_logout).setOnClickListener(new View.OnClickListener() {
+        rootView.findViewById(R.id.row_wpcom_connect).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 if (mAccountStore.hasAccessToken()) {
```

Comment:
We should change this to "If there is at least one local draft on one of their site then { show warning }" (iterate over sites / mPostStore.getPostsForSite - might be worth adding a new getter to the PostStore `hasLocalChangesOrLocalDrafts(site)` cc @aforcier )